core/src/main/java/org/apache/spark/shuffle/sort/ShuffleExternalSorter.java

```diff
@@ -22,6 +22,7 @@
 import java.io.IOException;
 import java.util.LinkedList;
 
+import org.apache.spark.storage.FileSegment;
```

Comment:
This import needs to be placed a few lines down with the other Spark imports on line 41; as-is, Scalastyle is going to complain about this.